static/redesign/apps/gallery/templates/form-item.html

```diff
@@ -1 +1 @@
-<a href="#" class="edit-form">{{ id }}: {{ name }}</a>
\ No newline at end of file
+<p class="edit-form">{{ name }}</p>
```

Comment:
Actually, nevermind! I think the CSS didn't pull down correctly when I was testing. Looks great! Merging!